configure.ac

```diff
@@ -302,7 +303,7 @@ AC_ARG_WITH([odoc],
 
 AC_ARG_ENABLE([ocamltest],
   [AS_HELP_STRING([--disable-ocamltest],
-    [do not build the ocamltest driver])])
+    [do not build the ocamltest test driver])])
```

Comment:
Oh dear, this message (written by me in #9250) was pretty dreadful. I think adding the word test makes it more confusing - could it more simply just be "do not build ocamltest"?